docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -962,3 +965,56 @@ To remove a key, such as channels, and all of its values:
 To configure channels and their priority for a single
 environment, make a ``.condarc`` file in the :ref:`root directory
 of that environment <config-channels>`.
+
+.. _threads:
+
+Configuring number of threads
+=============================
+
+You can use your ``.condarc`` file or environment variables to
+add configuration to control the number of threads. You may
+want to do this to tweak conda to better utilize your system.
+If you have a very fast SSD, you might be able to shorten the
+time it takes for conda to create environments and
+install/remove packages.
```

Comment:
I think you should add something like; users will be able to do this by increasing the number of threads.